pkg/operator/ceph/cluster/rbd/controller.go

```diff
@@ -60,7 +60,7 @@ var objectsToWatch = []client.Object{
 	&appsv1.Deployment{TypeMeta: metav1.TypeMeta{Kind: "Deployment", APIVersion: appsv1.SchemeGroupVersion.String()}},
 }
 
-var cephRBDMirrorKind = reflect.TypeOf(cephv1.CephRBDMirror{}).Name()
+var cephRBDMirrorKind = reflect.TypeFor[cephv1.CephRBDMirror]().Name()
```

Comment:
The same pattern also applies to all of these vars. Could you make the same changes for all of them?